tasks/fix-cat2.yml

```diff
@@ -726,7 +726,12 @@
       - RHEL-07-030320
 
 - name: "MEDIUM | RHEL-07-030321 | PATCH | The audit system must take appropriate action when there is an error sending audit records to a remote system."
-  command: "true"
+  lineinfile:
+      state: present
+      dest: /etc/audisp/audisp-remote.conf
+      regexp: '^network_failure_action = '
```

Comment:
Make sure to also searc hfor uncommented string.

Regexp: '^#?netowkr_failure_action'